bin/publish

```diff
@@ -43,4 +43,8 @@ if [ "$GIT_DESCRIPTION" = "v${VERSION}" ]; then
     echo 'Failed to log in to scan.connect.redhat.com'
     exit 1
   fi
+elif [ "$BRANCH_NAME" = "master" ]; then
```

Comment:
Its not relevant for redhat registry?

Reply:
Good point but I don't think so. I don't know if we would want to be pushing edge tags to RH because the publishing of tags is not an automated process. Our current policy for RH registry is that someone needs to push "publish" for tags to be public facing